pkg/loop/internal/relayer/providerext/mercury/mercury_reporting.go

```diff
@@ -10,22 +10,23 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/protobuf/types/known/emptypb"
 
+	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/core"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/net"
 	"github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb"
 	mercurypb "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/mercury"
 )
 
-type mercuryPluginFactoryClient struct {
+type PluginFactoryClient struct {
 	*net.BrokerExt
-	*ServiceClient
+	*core.ServiceClient
```

Comment:
Should we unexport these fields since the struct is now exported?

Reply:
yes, there is cleanup work to unexport both fields, structs and funcs.

i would prefer to do this as a second pass and use the PR to align on the pkg names and organization

i filed a ticket here, which includes my reasoning for doing this as a second pass (which i would start immediately after this is mergd) https://smartcontract-it.atlassian.net/browse/BCF-3114

Reply:
I didn't notice these were all still internal too, so I'm not actually concerned and follow up sounds good :+1: